joshua/joshua_model.py

```diff
@@ -694,6 +698,7 @@ def should_run_ensemble(tr: fdb.Transaction, ensemble_id: str) -> bool:
                 )
             )
 
+            _decrement(tr, ensemble_id, "started")
```

Comment:
If many agents are trying to steal this ensemble at the same time, do they likely conflict to each other and keep retrying? (We have read conflict on `started`)
How can we make sure the first agent gets it?

Reply:
Also, `should_run_ensemble` does not start a test. Meaning, when one agent decrements `started`, other agents can still get the same ensemble through `should_run_ensemble` before it gets picked up.

Reply:
One of them will get it, and the rest will fail with a conflict (and won't see any work to do when they retry)